### ISSUE TITLE
GB-2448808-A exists in client

### ISSUE THREAD

**Issue by madgpap** [j-siddle/surechembl-data-client#5]
although only WO, EP, US and JP are annotated - maybe byproduct of test?
See also:
https://www.surechembl.org/document/GB-2448808-A

**Comment by madgpap** [comment]
the same for:
https://www.surechembl.org/document/GB-2448807-A

**Comment by j-siddle** [comment]
Yes, I do remember processing one GB document to prove that arbitrary
documents would be processed by the TextAnnotator.

On 19 March 2015 at 17:14, George Papadatos <notifications@github.com>
wrote:

> the same for:
> https://www.surechembl.org/document/GB-2448807-A
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/j-siddle/surechembl-data-client/issues/5#issuecomment-83671916>
> .
>



-- 
email: jim@jamessiddle.net
web: www.jamessiddle.net

"There's an old saying in Tennessee -- I know it's in Texas, probably in
Tennessee -- that says, fool me once, shame on --shame on you. Fool me --
you can't get fooled again." - George W. Bush
